.pipelines/ci.yml

```diff
@@ -262,6 +264,144 @@ stages:
+              . ./hack/e2e/run-rp-and-e2e.sh
+              az cosmosdb sql database delete --name "$DATABASE_NAME" --yes --account-name "$DATABASE_ACCOUNT_NAME" --resource-group "$RESOURCEGROUP"
+            displayName: Clean Up Database
+            condition: always()
+
+          # AZ CLI Logout
+          - template: ./templates/template-az-cli-logout.yml
+      - job: Run_E2E_Tests_For_MIWI
+        timeoutInMinutes: 0
+        pool:
+          name: 1es-aro-ci-pool
+        steps:
+          # Checkout the code
+          - template: ./templates/template-checkout.yml
+
+          # Install Docker, Docker Compose, and dependencies
+          - bash: |
+              . ./hack/e2e/utils.sh
+              install_docker_dependencies
+            displayName: Install Docker and Docker Compose
+          - task: Docker@2
+            inputs:
+              command: "login"
+              containerRegistry: arointsvc #service connection name
+
+          # AZ CLI Login
+          - template: ./templates/template-az-cli-login.yml
+            parameters:
+              azureDevOpsJSONSPN: $(aro-v4-e2e-devops-spn)
+
+          - bash: |
+              az account set -s $AZURE_SUBSCRIPTION_ID
+              SECRET_SA_ACCOUNT_NAME=$(SECRET_SA_ACCOUNT_NAME) make secrets
+            displayName: Fetch secrets
+
+          - bash: |
+              echo "##vso[task.setvariable variable=CI]true"
+            displayName: Set CI=true
+
+          # Override the E2E label for IndividualCI/BatchedCI (i.e. not manually
+          # ran/PR jobs) to run all non-smoke tasks (default is !smoke&&!regressiontest)
+          - bash: |
+              echo "##vso[task.setvariable variable=E2E_LABEL]!smoke"
+            displayName: Enable regression tests in CI
+            condition: in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')
+
+          # Get Kubeconfig for AKS Cluster with corrected Key Vault configuration
+          - bash: |
+              . secrets/env
+
+              # Retrieve the kubeconfig
+              hack/get-admin-aks-kubeconfig.sh > aks.kubeconfig
+
+              if [ -f aks.kubeconfig ]; then
+                echo "Kubeconfig retrieved successfully."
+              else
+                echo "Failed to retrieve Kubeconfig."
+                exit 1
+              fi
+            displayName: Get Kubeconfig for AKS Cluster
+
+          # Run the E2E test suite
+          - bash: |
+              echo "##vso[task.setvariable variable=USE_WI]true"
+            displayName: Set USE_WI=true
+          - bash: |
+              . secrets/env
+              . ./hack/e2e/run-rp-and-e2e.sh
+              echo "##vso[task.setvariable variable=CLUSTER]${CLUSTER}"
+              echo "###vso[task.setvariable variable=DATABASE_NAME]{$DATABASE_NAME}"
+              export SKIP_MIWI_ROLE_ASSIGNMENT=true
+              . ./hack/devtools/local_dev_env.sh
+              create_miwi_env_file
+              echo "created miwi env file successfully"
```

Comment:
I think this log line would log even if the `create_miwi_env_file` was unsuccessful, if we need this log line, we should ensure that it's only ran if we get a successful exit code from `create_miwi_env_file`. Or for simplicity's sake we could also just remove this log line.